Assets/tiny_rogue/Scripts/TinyRogueConstants.cs

```diff
@@ -5,8 +5,8 @@ namespace game
 {
     public class TinyRogueConstants
     {
-        public static float TileWidth = 0.09f;
-		public static float TileHeight = 0.16f;
+        public static float TileWidth = 1f;
```

Comment:
This needs to be defined by whether we're ascii or not, somehow

Reply:
Yes this would break the ASCII version at the moment as the tiles are none square and different sizes. Can we use a #ifdefine?

Reply:
There's no if define, just `GlobalGraphicsSettings.ascii` as a global static

Reply:
Then I guess they can be changed to get functions with that as a runtime check.

Reply:
should we put these constants on a component like graphicssettings instead?